### PR TITLE
Fix page transitions bug

### PR DESCRIPTION
**Pull request by EseCoder**
## Bug Fix

**Description:**
This commit fixes an activity transition problem that affects the UX of the app. Whenever a user resumes using the app a lock screen appears to Auth the user in but after successful auth, it doesn't take them back to the previous page or activity they were on.
These changes address that by (1.) not finishing the previous activity in the first place. (2.) removing any activity starting intent flags that might try and destroy other activities started before it.

**Select the type of change(s) made in this pull request:**
- [x] Bug fix *(Fixes an issue)*
- [ ] New feature *(Adds functionality)*
- [ ] Documentation *(Fix to documentation)*

----------------------------------------------------------------------------------------

Fixes #issue-number
https://chat.wearehorizontal.org/boards/workspace/ztai374a6pbr8n4a7nzdsxa5qr/bw5m9wktjkpr75n5w7n4r3yknmr/vk8nh34y48ty33jwsumzyxe76ch/c84jmeqbzrir1jpkeeoe1tp1ety

## Proposed changes 

* These changes address that by (1.) not finishing the previous activity in the first place. (2.) removing any activity starting intent flags that might try and destroy other activities started before it.
*
*

### CODE REVIEW

File: mobile/src/main/java/rs/readahead/washington/mobile/views/base_ui/MetadataBaseLockFragment.kt [hunk 1]
```diff
@@ -40,7 +40,7 @@ open class MetadataBaseLockFragment  : Fragment(){
         super.onViewCreated(view, savedInstanceState)
     }
     override fun onResume() {
-        activity.restrictActivity()
+        //activity.restrictActivity()
```
Comment (ahlem-jarrar) [inline mobile/src/main/java/rs/readahead/washington/mobile/views/base_ui/MetadataBaseLockFragment.kt:43]: This should not be removed, it make sure that the user is not using the app without an unlocked key store.

Comment (EseCoder) [reply mobile/src/main/java/rs/readahead/washington/mobile/views/base_ui/MetadataBaseLockFragment.kt:43]: This is called already from MainActivity on which MicFragment is a part of. That's why I said it launches a double lock screen.
Except if we want a double lock screen.

File: mobile/src/main/java/rs/readahead/washington/mobile/views/settings/SecuritySettings.kt [hunk 2]
```diff
@@ -240,7 +240,7 @@ class SecuritySettings : BaseFragment() {
         intent!!.putExtra(RETURN_ACTIVITY, returnCall.getActivityOrder())
         intent.putExtra(IS_FROM_SETTINGS, true)
         startActivity(intent)
-        activity.finish()
+        //activity.finish()
```
Comment (ahlem-jarrar) [inline mobile/src/main/java/rs/readahead/washington/mobile/views/settings/SecuritySettings.kt:243]: I think if we just want the user to came back to the security settings we need just not finish the current activity..

**Review by ahlem-jarrar — COMMENTED** [review]
Please do not delete the .gitignore file and push it ..

**Pull request closed without merging**